src/dafny/NonlinearArithmetic/README.md

```diff
@@ -0,0 +1,9 @@
+# Overview
```

Comment:
This text should presumably be migrated to `NonlinearArithmetic.md`.

Reply:
Guess it was easy enough to write that file.